src/main/java/wolox/training/models/Book.java

```diff
@@ -150,6 +160,7 @@ public long getId() {
     }
 
     public void setId(long id) {
```

Comment:
this set is not necessary because the id is autoincremental

Reply:
I removed the set method